### PR TITLE
chore: remove website

### PR DESCRIPTION
**Pull request by jooola**
Please don't squash

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -51,5 +52,27 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/checkout@v3
+        with:
+          repository: libretime/website
+          path: website
+          # TODO: Rename or use dedicated deploy key
```
[the remark this answers is not in the document]
Comment (paddatrapper) [reply .github/workflows/docs.yml:74]: Sure, can set up another once

**Comment by paddatrapper** [comment]
I have switched over https://libretime.org. There may be some downtime while it migrates, but it should not take long

File: .github/workflows/docs.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (paddatrapper) [reply .github/workflows/docs.yml:74]: I've created `WEBSITE_DEPLOY_KEY` for this. I think we should not re-use these, as a compromised key can then write to all our additional repositories. They need to be added to the repositories individually anyway

[the remark this answers is not in the document]
Comment (jooola) [reply .github/workflows/docs.yml]: Done

**Review by paddatrapper — APPROVED** [review]

**Pull request merged**